lab_6_pipeline/pipeline.py

```diff
@@ -4,15 +4,38 @@
 # pylint: disable=too-few-public-methods, unused-import, undefined-variable, too-many-nested-blocks
 import pathlib
 
-try:
-    from networkx import DiGraph
-except ImportError:  # pragma: no cover
-    DiGraph = None  # type: ignore
-    print('No libraries installed. Failed to import.')
-
-from core_utils.article.article import Article
+import spacy
+import spacy_udpipe
+import stanza
+from networkx import DiGraph
+from stanza.models.common.doc import Document
+from stanza.pipeline.core import Pipeline
+from stanza.utils.conll import CoNLL
+
+from core_utils.article.article import Article, ArtifactType, get_article_id_from_filepath
+from core_utils.article.io import from_meta, from_raw, to_cleaned, to_meta
+from core_utils.constants import ASSETS_PATH, UDPIPE_MODEL_PATH
 from core_utils.pipeline import (AbstractCoNLLUAnalyzer, CoNLLUDocument, LibraryWrapper,
                                  PipelineProtocol, StanzaDocument, TreeNode)
+from core_utils.visualizer import visualize
+
+
+class EmptyDirectoryError(Exception):
+    """
+    Raising an error when the directory is empty
+    """
+
+
+class InconsistentDatasetError(Exception):
+    """
+    IDs contain slips, number of meta and raw files is not equal, files are empty;
+    """
+
+
+class EmptyFileError(Exception):
+    """
+    I don't khow for what
```

Comment:
?

Reply:
Sorry, I just forgot to change, but it was necessary to create it before usage, so I didn't know when it will be used